script/Ciel/Day 14/14_01_CIEL14_1.txt

```diff
@@ -486,17 +527,27 @@ How many seconds? How many minutes... will this... pain... go on... for?
 [sha:c082dfd9b9823b6d3670c48016083d0ed4ce8f53]{
 -- Page 29, Offset 37100.
 -- いたい。どうにかしようと、ナイフで掌をザクザクと刺してみた。ダメだ。ぜんぜんイタク、ない。そうか。たぶんきっと線の　セイだ。眼鏡。眼鏡をかければ、いつものように、こんな異常は、収まって
-It hurts. In one way or another, I try to stab away at my palm with my knife. It's no good. It doesn't hurt at all. I see. It must be because of the lines. My glasses. If I put on my glasses, this kind of abnormality will go away, as always
+It hurts.
+ In a desperate attempt to do %{i}something%{/i}, I stab the palm of my hand over and over again with my knife.
+ It's no use.
+ It doesn't hurt at all.
+ Wait... of course.
+ This is all because of the lines.
+ My glasses――this will all go away if I put on my glasses, just like it always does.
 }
 [sha:47db2eeffcce842c8ac7283706f2b9acc8cfdc48]{
 -- Page 29, Offset 37101.
 -- 「―――かけ、てる。……かけてる。かけてる―――！」
-"―――It's... on...... It's on. It's on―――!"
+"I'm... wearing them... I'm wearing them, I'm already wearing them―――!"
 }
 [sha:62639c29c5383aedf54c97ef33abe74053b083c9]{
 -- Page 30, Offset 37102.
 -- だめだ。わらうな。わらえば、タガがハズれる。でもわらいたくて仕方がない。だってこれじゃ、もうどうしようもない。
-I can't. Don't laugh. If I do, I'll lose control. But I can't help but want to laugh. I mean, right now, there's nothing else left.
+No.
+ I mustn't laugh.
+ If I do, I'll lose control.
+ But I can't help but want to laugh.
+ I mean, what else can I do at this point?
```

Comment:
>Stop.
I can'  t   la u gh.
If  I　do, I    w il l   lo se  co ntr ol.
But how can I not?
I  mea n,   wh a t   els e　ca n  I   d o?